packages/core/src/data-manager/TransportManager.ts

```
@@ -59,7 +59,7 @@ export default class TransportManager {
       } else {
         await this.transport.init(HttpLogger);
       }
-      Log.debug('Configuring transports');
+      Log.debug('Configuring transports', this.messageVersion);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)